FAQ.md

```diff
@@ -0,0 +1,125 @@
+2. Add the release to the list at [pipeline file](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines/build)
+3. Adjust the PR testing pipline [Example](https://github.com/AdoptOpenJDK/openjdk-build/pull/1394) to use the new release
+
+## Removing a major release once you've added a new one
+
+Unless the last release was an LTS one, you will generally want to remove one of the old versions after creating a new one. This can be done with `disableJob = true` in the release configuration files
+
+[Example](https://github.com/AdoptOpenJDK/openjdk-build/pull/1303/files)
+
+## How to enable/disable a particular build configuration
+
+1. Add/Remove it from the configuration files in https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines/jobs/configurations
+2. if you're removing one and it's not just temporarily, you may want to delete the specific job from jenkins too
+
+[Example PR - removing aarch64 OpenJ9 builds](https://github.com/AdoptOpenJDK/openjdk-build/pull/1452)
+
+## How to add a new variant
+
+First you will need to add support into the [pipeline files](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines/build) as well as any environment-specific change syou need to make in the [platform files](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/build-farm/platform-specific-configurations)
+For an example, see [this PR where Dragonwell was added](https://github.com/AdoptOpenJDK/openjdk-build/pull/2051/files)
+For more information on other changes required, see [this document](https://github.com/AdoptOpenJDK/TSC/wiki/Adding-a-new-build-variant)
+
+## How do I change the parameters, such as configure flags, for a jenkins build
+
+Either
+- Modify the environment files in [platform-specific-configurations](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/build-farm/platform-specific-configurations)
+- Modify the [pipeline files](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines/build), although this is normally only done for configuration differences such as OpenJ9 Large Heap builds
+
+[Example PR - Adding a new configure flag for OpenJ9](https://github.com/AdoptOpenJDK/openjdk-build/pull/1442/files)
+
+## How to do a new release build
+
+   Since it's quite long, this is covered in a separate [RELEASING.md](RELEASING.md) document
+
+## I've modified the build scripts - how can I test my changes?
+
+If you're making changes ensure you follow the contribution guidelines in
+[CONTRIBUTING.md](CONTRIBUTING.md) including running `./shellcheck.sh` if you're modifying
+the shell scripts.
+
+In order to test whether your changes work use the
+[test-build-script-pull-request](https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/test-build-script-pull-request/)
+job! Pass it your fork name (e.g. https://github.com/sxa555/openjdk-build) and the name of the branch
+and it will run a build using your updated scripts.
```

Comment:
What permissions are required? I cannot see the options you're talking about.

Reply:
I think the default interface to that should probably be the "run tests" trigger (I don't think we've had that documented yet ...) but at the moment it is the jenkins system permissions which is very restrictive.

As part of the permissions overhaul this will be a good one to open up to more of the build team so we can revise this once those changes are put through.

Reply:
I'm working on writing up some documentation for the pr-tester as we speak. I should have that in within the hour

Reply:
We'll add that in once Morgan's new doc is in :-)